server/mergin/sync/models.py

```diff
@@ -833,7 +834,71 @@ def files(self) -> List[ProjectFile]:
             WHERE fh.change != 'delete';
         """
         params = {"project_id": self.project_id, "version": self.name}
-        result = db.session.execute(query, params).fetchall()
+        return db.session.execute(query, params).fetchall()
```

Comment:
Is this due to the fact that we now return early whereas before we did something else perhaps with result and returned it later?